pkg/middlewares/auth/forward.go

```diff
@@ -88,6 +88,14 @@ func (fa *forwardAuth) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	tr, err := tracing.FromContext(req.Context())
```

Comment:
I think if there is not tracing in the context we should not start a new trace. We should only honor those traces the come from traefik itself.

Reply:
If there is no tracing in context, then error will be returned  (https://github.com/containous/traefik/blob/54bc758463db39bd227046d6584c49ff57da24d5/pkg/tracing/tracing.go#L36) and `forward-auth` span will not be started